code/modules/culture_descriptor/location/locations_human.dm

```diff
@@ -121,17 +121,28 @@
 
 /decl/cultural_info/location/human/terra
 	name = HOME_SYSTEM_TERRA
-	description = "Terra, in the Gilgamesh system. Is not the planet Earth as some may be lead to believe. No, Terra is the home of the Gilgamesh Colonial Confederation, and the capital of that \
-	particular superpower. Terra is one of the many affiliated planets in the GCC and the one that is perhaps the most important. Terra, much like most of the GCC, is isolationist, with a minimal \
-	non-human presence. Much of Terra is heavily industrialised, with a lower standard of living than on equivalent SCG planets. Much of the population works in services, industry or the military. \
-	Still recovering from an economic crash toward the end of the Gaia conflict, much of Terra is in decline, with public services and works being minimal and much of the population burdened by \
-	increasingly high taxes. This state of affairs has seen the immigration of many Terran citizens to the SCG in recent years, hoping to escape the poor conditions they found themselves in. Despite this,\
-	the people of Terra are a proud and unbroken people who are heavily patriotic and tend to bring this with them wherever they go."
+	description = "Terra - the capital world of Terran Government, planet with an atmosphere close to that of Earth and home to many 'true' humans. \
+	Almost 100% of its population is humans, with rare exception for few Skrell ambassadors, which are arguably the only non-human species that are allowed on Terra. \
```

Comment:
[...] is human, with the rare exception of a few Skrell ambassadors [...]
[...] the only non-human species that is allowed [...]